src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -1211,7 +1211,7 @@ namespace HeatBalanceSurfaceManager {
                         }
                     }
                 // interior window report
-                }else if (((Surface(iSurf).Class == SurfaceClass_Window) || (Surface(iSurf).Class == SurfaceClass_TDD_Dome)) && (!state.dataConstruction->Construct(Surface(iSurf).Construction).TypeIsAirBoundaryInteriorWindow)) {
+                }else if ((Surface(iSurf).Class == SurfaceClass_Window) || (Surface(iSurf).Class == SurfaceClass_TDD_Dome)) {
```

Comment:
Missing piece of whitespace at the beginning here, just to note for a future commit.  (_Comment made mostly to prove that I read all the changes since I don't have many comments yet._)

Reply:
OK, will reapply formatting in this section.